v1/pipeline/pipeline_using_template_spec.yaml

```diff
@@ -0,0 +1,27 @@
+type: object
+title: PipelineUsingTemplateSpec
+required:
+  - id
+  - versionLabel
+properties:
+  id:
+    type: string
+    description: id of template to be linked in pipeline
+    pattern: "^[a-zA-Z_][0-9a-zA-Z_]{0,127}$"
+  versionLabel:
```

Comment:
convert t snake case or even better can we call this label?

Reply:
isnt label generic and may get confused with labels of template? if we introduce in future? so should we go ahead with version_label?

Reply:
can we adopt this to the ref itself?  id@version

like my_template@v1.0

Reply:
+1

Reply:
updated schema to use ref keyword with value as id@version